lms/djangoapps/support/views/manage_user.py

```diff
@@ -18,7 +18,7 @@
 from openedx.core.djangoapps.user_api.accounts.serializers import AccountUserSerializer
 from openedx.core.djangolib.oauth2_retirement_utils import retire_dot_oauth2_models
 
-from edx_django_utils.user import generate_password  # lint-amnesty, pylint: disable=wrong-import-order
+from eox_nelp.user_authn.utils import generate_password  # lint-amnesty, pylint: disable=wrong-import-order
```

Comment:
Thanks @johanseto. Please check the comment on the original Palm upgrade pull request here:

 - https://github.com/eduNEXT/eox-nelp/pull/130/files#r1686935870

Let me know what do you think.